File: app/src/main/java/com/jumi/shareproject/MainActivity.java
package com.jumi.shareproject;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.jumi.shareproject.compoment.ShTitleBar;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        ShTitleBar titleBar = new ShTitleBar(this);
        setContentView(titleBar);
    }
}

File: app/src/main/java/com/jumi/shareproject/MainActivity.kt
package com.jumi.shareproject

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import com.dwwd.dwlibrary.compoment.ShTitleBar;

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val titleBar = ShTitleBar(this)
        setContentView(titleBar)
    }
}
